Packs/PaloAltoNetworks_IoT/Integrations/PaloAltoNetworks_IoT/PaloAltoNetworks_IoT.yml

```diff
@@ -29,16 +29,30 @@ configuration:
   name: proxy
   required: false
   type: 8
+- defaultvalue: 2020-01-01T00:00:00Z
+  display: First fetch time
+  name: first_fetch
+  required: false
+  type: 0
+  additionalinfo: First fetch time in the ISO format - YYYY-MM-DDTHH:mm:ssZ or -1 if fetching from the beginning
```

Comment:
We need to support the format of `<number> <time unit>`. e.g., 12 hours, 7 days, 2 seconds etc.
like I wrote here https://github.com/demisto/content/pull/8037#discussion_r462863803

Reply:
Have to `.replace(tzinfo=timezone.utc)` before the timestamp() call since timestamp() is time zone. We want everything to be "UTC" everywhere.